onfido-java/src/main/java/com/onfido/EvidenceFileManager.java

```diff
@@ -0,0 +1,19 @@
+package com.onfido;
+
+import com.onfido.api.Config;
+import com.onfido.api.FileDownload;
+import com.onfido.api.ResourceManager;
+import com.onfido.exceptions.OnfidoException;
+import okhttp3.OkHttpClient;
+
+public class EvidenceFileManager extends ResourceManager {
```

Comment:
Hello again. Sorry for the back and forth but the addition of the evidence file endpoint must be skipped in this PR. We (Onfido) are reworking this product and clients shouldn't integrate with the current implementation. Therefore, we kindly request you to remove this and only keep the `Tasks` endpoints.